doc/guide/cockpit-util.xml

```diff
@@ -40,23 +40,66 @@ array = cockpit.format_bytes(number, [factor, separate])
 
     <para>If the <code>number</code> is less than the <code>factor</code> or an unknown factor
       was passed in, then the formatted number is returned without a suffix. If <code>separate</code>
-      is true, returns an array of <code>[formatted_number, suffix]</code> unless no suffix is returned.</para>
+      is true, returns an array of <code>[formatted_number, suffix]</code> or
+      <code>[formatted_number]</code> if returned without a suffix.</para>
+
+    <para>If <code>number</code> is <code>null</code> or <code>undefined</code> an empty string or array
+      will be returned.</para>
+  </refsection>
```

Comment:
I think an empty array is problematic since the existing clients don't handle it.  They will put `result[0]` on the screen, for example, which is "undefined", no?